projects/storefrontapp-e2e-cypress/cypress/helpers/product-configurator-conflict-dialog.ts

```diff
@@ -0,0 +1,136 @@
+/*
+ * SPDX-FileCopyrightText: 2023 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as configuration from './product-configurator';
+import * as configurationVc from './product-configurator-vc';
+
+/**
+ * verifies that the conflict solver dialog is currently opened
+ */
+export function checkIsOpen() {
+  cy.get('cx-configurator-conflict-solver-dialog').should('be.visible');
+}
+
+/**
+ * verifies that the conflict solver dialog is currently closed
+ */
+export function checkIsClosed() {
+  cy.get('cx-configurator-conflict-solver-dialog').should('not.exist');
+}
+
+/**
+ * verifies the content of the conflict solver dialog
+ * @param involvedAttributes expected involved attributes for the currently visible conflict
+ */
+export function checkDisplayedConflict(
+  involvedAttributes: {
+    name: string;
+    selectedValueNames: string[];
+    uiType: configuration.uiType;
+  }[]
+) {
+  checkHasTitle();
+  checkHasCloseButton();
+  checkHasResolveMessage();
+  checkHasDescription();
+  checkNumberOfSuggestions(involvedAttributes.length);
+  involvedAttributes.forEach((attr) =>
+    checkAttributeInvolved(attr.name, attr.selectedValueNames, attr.uiType)
+  );
+}
+
+/**
+ * verifies that the conflict solver dialog has a title
+ */
+export function checkHasTitle() {
+  cy.get('.cx-dialog-title').should('be.visible');
+}
+
+/**
+ * verifies that the conflict solver dialog has a close button
+ */
+export function checkHasCloseButton() {
+  cy.get('button.close').should('be.visible');
+}
+
+/**
+ * verifies that the conflict solver dialog displays the resole issues message
+ */
+export function checkHasResolveMessage() {
+  cy.get('.cx-dialog-body .cx-msg-warning').should('be.visible');
+}
+
+/**
+ * verifies that the conflict solver dialog displays the conflict description
+ */
+export function checkHasDescription() {
+  cy.get('cx-configurator-conflict-description').should('be.visible');
+}
+
+/**
+ * verifies that the conflict solver dialog displays the given number of suggestions
+ * @param {number} number number of expected suggestions
+ */
+export function checkNumberOfSuggestions(number: number) {
```

Comment:
@Uli-Tiger, could you rename a parameter? I wonder why it works. In Java it is not allowed to use the same name.
What about `amount`?